.circleci/config.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
           name: unit tests
-          command: npm run test
+          command: npm run test:ci
```

Comment:
this run the tests in line - slower than in development but still much faster than before